examples/ustream/ustream_client.c

```diff
@@ -90,7 +90,7 @@ int main(int argc, FAR char *argv[])
 
   printf("client: Connecting to %s...\n", CONFIG_EXAMPLES_USTREAM_ADDR);
   addrlen += sizeof(sa_family_t) + 1;
-  ret = connect( sockfd, (struct sockaddr *)&myaddr, addrlen);
+  ret = connect(sockfd, (struct sockaddr *)&myaddr, addrlen);
```

Comment:
```suggestion
  ret = connect(sockfd, (FAR struct sockaddr *)&myaddr, addrlen);
```

Reply:
since the whole ustream_client.c doesn't add FAR, let's do the clean up in the new PR instead.